pkg/validator/validator_test.go

```diff
@@ -39,9 +39,11 @@ func TestValidation(t *testing.T) {
 		{
 			name: "happy_path",
 			issuesHandler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				w.WriteHeader(http.StatusOK)
```

Comment:
You shouldn't need this - the default behavior writes a 200

Reply:
Good to know, cleaned them up now :D